### DISCUSSION TITLE
Idea: Show users on world map

### DISCUSSION THREAD

**Discussion by jschuur** [jschuur/coworking.party#83] [Ideas]
To make users feel part of a global community, show their location on a map or globe.

This should at least allow opting out of and could use [CF headers](https://github.com/jschuur/coworking.party/issues/19) to identify the country/city.

Opportunity here to use the [Twitch globe](https://twitch-globe.loki.m4p.uk/) codebase that @DR-DinoMight is working on?

If a user has opted in, there should be a reminder when this data is publicly shared, so they can change their mind again.

**Comment by mdeloughry** [comment]
Personally I don't like this would be an opt-out feature, when I built the twitch globe that why I made it free text so you can be as narrow or as wide as you'd like. 

Ideally I'd like to see a profile setting where a user can put in their location, and you do the geoip lookup like it do and use [Nominatim](https://nominatim.org/release-docs/latest/api/Search/) based on the value they provide, if none is present they don't show up on the map.

I think if it's going to be a opt-out situation, you keep it anonymous and just show a point instead of labels.


But I'm happy to look at building some form of comms, that you can use to communicate with a special CWP globe.

**Comment by jschuur** [comment]
> Ideally I'd like to see a profile setting where a user can put in their location, and you do the geoip lookup like it do and use [Nominatim](https://nominatim.org/release-docs/latest/api/Search/) based on the value they provide, if none is present they don't show up on the map.

Search based is interesting, since it gives people the flexibility to be as broad or narrow as they want perhaps (region/city/borough e.g.)
